src/App.js

```diff
@@ -8,6 +9,7 @@ function App() {
         Read through the instructions in the README.md file to build your NASA
```

Comment:
We just forgot to get rid of this particular line which shows up at the top of the page, but no biggie.